src/back/space/src/repo/table_space_user.cpp

```diff
@@ -0,0 +1,174 @@
+#include "table_space_user.hpp"
+#include "../../../shared/errors.hpp"
+#include "../../../shared/paging.hpp"
+#include <chrono>
+
+#include <userver/components/component_config.hpp>
+#include <userver/components/component_context.hpp>
+#include <userver/utils/boost_uuid4.hpp>
+#include <userver/yaml_config/merge_schemas.hpp>
+#include <userver/storages/postgres/component.hpp>
+
+namespace svetit::space::table {
+
+namespace pg = storages::postgres;
+using pg::ClusterHostType;
+
+SpaceUser::SpaceUser(pg::ClusterPtr pg)
+	: _pg{std::move(pg)}
+{
+	//InsertDataForMocks();
+}
+
+const pg::Query kInsertSpaceUser{
+	"INSERT INTO space.user (spaceId, userId, isOwner, role) "
+	"VALUES ($1, $2, $3, $4) ",
+	pg::Query::Name{"insert_space_user"},
+};
+
+void SpaceUser::Insert(
+	const boost::uuids::uuid& spaceId,
+	const std::string& userId,
+	bool isOwner,
+	Role::Type role)
+{
+	_pg->Execute(ClusterHostType::kMaster, kInsertSpaceUser, spaceId, userId, isOwner, role);
+}
+
+const pg::Query kDeleteBySpace {
+	"DELETE FROM space.user WHERE spaceId = $1",
+	pg::Query::Name{"delete_user_by_space"},
+};
+
+void SpaceUser::DeleteBySpace(const boost::uuids::uuid& spaceUuid) {
+	auto res = _pg->Execute(ClusterHostType::kMaster, kDeleteBySpace, spaceUuid);
+	if (!res.RowsAffected())
+		throw errors::NotFound();
+}
+
+const pg::Query kIsOwner {
+	"SELECT isOwner FROM space.user WHERE spaceId = $1 AND userId = $2",
+	pg::Query::Name{"is_owner"},
+};
+
+bool SpaceUser::IsOwner(const boost::uuids::uuid& spaceUuid, const std::string& userId) {
```

Comment:
Давай пожалуйста везде для сущностей где поле с типом UUID является первичным ключом, переменные называть id, а не uuid
```suggestion
bool SpaceUser::IsOwner(const boost::uuids::uuid& spaceId, const std::string& userId) {
```